"Go: master Xcode: xcode6.4" CI fails, apparently during graphviz install

Saw this in the travis check for #254.

```
Updating Homebrew...
==> Installing dependencies for graphviz: libtool, pkg-config, libpng, freetype, fontconfig, jpeg, libtiff, webp, gd
==> Installing graphviz dependency: libtool
==> Downloading https://homebrew.bintray.com/bottles/libtool-2.4.6_1.yosemite.bo
==> Pouring libtool-2.4.6_1.yosemite.bottle.tar.gz
==> Caveats
In order to prevent conflicts with Apple's own libtool we have prepended a "g"
so, you have instead: glibtool and glibtoolize.
==> Summary
🍺  /usr/local/Cellar/libtool/2.4.6_1: 70 files, 3.7MB
==> Installing graphviz dependency: pkg-config
==> Downloading https://homebrew.bintray.com/bottles/pkg-config-0.29.2.yosemite.
==> Pouring pkg-config-0.29.2.yosemite.bottle.tar.gz
🍺  /usr/local/Cellar/pkg-config/0.29.2: 11 files, 631KB
==> Installing graphviz dependency: libpng
==> Downloading https://downloads.sourceforge.net/libpng/libpng-1.6.34.tar.xz
==> Downloading from https://downloads.sourceforge.net/project/libpng/libpng16/1
==> ./configure --disable-silent-rules --prefix=/usr/local/Cellar/libpng/1.6.34
==> make
==> make test
Last 15 lines from /Users/travis/Library/Logs/Homebrew/libpng/03.make:
============================================================================
# TOTAL: 33
# PASS:  32
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to png-mng-implement@lists.sourceforge.net
============================================================================
make[2]: *** [test-suite.log] Error 1
make[1]: *** [check-TESTS] Error 2
make: *** [check-am] Error 2
Do not report this issue to Homebrew/brew or Homebrew/core!
These open issues may also help:
Formulae with opportunistic linkage issues https://github.com/Homebrew/homebrew-core/issues/16531
php: 7.1 (new formula) https://github.com/Homebrew/homebrew-core/pull/16067
Error: You are using macOS 10.10.
We (and Apple) do not provide support for this old version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.
Error: You are using macOS 10.10.
We (and Apple) do not provide support for this old version.
You may encounter build failures or other breakages.
Please create pull-requests instead of filing issues.
The command "if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install graphviz; fi" failed and exited with 1 during .
Your build has been stopped.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,6 +2,8 @@ language: go
 
 go_import_path: github.com/google/pprof
 
+env: SKIP_GRAPHVIZ=0
+
 matrix:
   include:
     - os: linux
@@ -15,12 +17,15 @@ matrix:
     - os: osx
       osx_image: xcode6.4
       go: 1.8.x
+      env: SKIP_GRAPHVIZ=1
     - os: osx
       osx_image: xcode6.4
       go: 1.9.x
+      env: SKIP_GRAPHVIZ=1
     - os: osx
       osx_image: xcode6.4
       go: master
+      env: SKIP_GRAPHVIZ=1
     - os: osx
       osx_image: xcode7.3
       go: 1.8.x
@@ -47,8 +52,8 @@ addons:
       
 before_install:
   - go get -u github.com/golang/lint/golint honnef.co/go/tools/cmd/...   
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update          ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install graphviz; fi
+  - if (("$TRAVIS_OS_NAME" == "osx") && (!"$SKIP_GRAPHVIZ")); then brew update          ; fi
+  - if (("$TRAVIS_OS_NAME" == "osx") && (!"$SKIP_GRAPHVIZ")); then brew install graphviz; fi
 
 script:
   - gofmtdiff=$(gofmt -s -d .) && if [ -n "$gofmtdiff" ]; then printf 'gofmt -s found:\n%s\n' "$gofmtdiff" && exit 1; fi

